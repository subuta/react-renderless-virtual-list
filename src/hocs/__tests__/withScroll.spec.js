import React from 'react'
import { create } from 'react-test-renderer'

const Hoge = () => {
  return (
    <h1>hoge</h1>
  )
}

test('city database has Vienna', () => {
  const tree = create(<Hoge />).toJSON()
  expect(tree).toMatchSnapshot()
});
