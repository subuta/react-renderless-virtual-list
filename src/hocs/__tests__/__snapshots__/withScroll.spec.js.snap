// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`city database has Vienna 1`] = `
<h1>
  hoge
</h1>
`;
